week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -37,7 +55,21 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  let newarray = [];
+  let temp;
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] > 100) {
+      arr[i] = 100;
+      temp = (arr[i].toFixed(2) * 100) / 100;
+      newarray[i] = temp.toString() + "%";
+    } else {
+      temp = (arr[i].toFixed(2) * 100) / 100;
+      newarray[i] = temp.toString() + "%";
+    }
+  }
```

Comment:
It is a correct solution but there is some unnecessary code repetition there (DRY - don't repeat yourself)
It could be shortened to something like this
```
    if (arr[i] > 100) {
      arr[i] = 100;
    }
    temp = (arr[i].toFixed(2) * 100) / 100;
    newarray[i] = temp.toString() + "%";
```